build/patches/Disable-all-predictors-code.patch

```diff
@@ -175,22 +179,22 @@ diff --git a/components/optimization_guide/core/prediction_model_download_manage
      0x5c, 0x7f, 0xd0, 0x0c, 0xa1, 0x6e, 0xef, 0xdd, 0x63, 0x7a};
  const net::NetworkTrafficAnnotationTag
      kOptimizationGuidePredictionModelsTrafficAnnotation =
-@@ -106,6 +106,7 @@ PredictionModelDownloadManager::~PredictionModelDownloadManager() = default;
+@@ -111,6 +111,7 @@ base::FilePath::StringType PredictionModelDownloadManager::ModelInfoFileName() {
  void PredictionModelDownloadManager::StartDownload(
      const GURL& download_url,
      proto::OptimizationTarget optimization_target) {
 +  if ((true)) return;
    download::DownloadParams download_params;
    download_params.client =
        download::DownloadClient::OPTIMIZATION_GUIDE_PREDICTION_MODELS;
-@@ -240,6 +241,7 @@ void PredictionModelDownloadManager::OnDownloadFailed(
+@@ -247,6 +248,7 @@ void PredictionModelDownloadManager::OnDownloadFailed(
  absl::optional<std::pair<base::FilePath, base::FilePath>>
- PredictionModelDownloadManager::ProcessDownload(
-     const base::FilePath& file_path) {
+ PredictionModelDownloadManager::VerifyDownload(const base::FilePath& file_path,
+                                                bool delete_file_on_error) {
 +  if ((true)) return absl::nullopt;
```

Comment:
i think file deletion is missing

```
@@ -247,6 +248,12 @@ void PredictionModelDownloadManager::OnDownloadFailed(
 absl::optional<std::pair<base::FilePath, base::FilePath>>
 PredictionModelDownloadManager::VerifyDownload(const base::FilePath& file_path,
                                                bool delete_file_on_error) {
+  if ((true)) {
+    base::ThreadPool::PostTask(
+        FROM_HERE, {base::TaskPriority::BEST_EFFORT, base::MayBlock()},
+        base::GetDeleteFileCallback(file_path));
+    return absl::nullopt;
+  }
   if (!switches::ShouldSkipModelDownloadVerificationForTesting()) {
     // Verify that the |file_path| contains a valid CRX file.
     std::string public_key;
```

Reply:
Why should it be deleted?

Reply:
We shouldn't even download it if we are going to delete it. I think it is not downloaded.

Reply:
> I think it is not downloaded.

sure, but it doesn't get there then and that `if ((true)) return absl::nullopt;` isn't necessary.
if it arrives there for some reason, the file is present and must be deleted